x/ccv/provider/keeper/grpc_query.go

```diff
@@ -67,8 +72,13 @@ func (k Keeper) QueryConsumerChainStops(goCtx context.Context, req *types.QueryC
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
+	// get all pending consumer removal proposals
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	props := k.GetAllConsumerRemovalProps(ctx)
+	props := types.ConsumerRemovalProposals{}
```

Comment:
Why not have a function `GetPendingConsumerRemovalProps`?

Reply:
What would be the benefit?

Reply:
Consistency and easier testing.

This is done inside a query  function - those have lots of moving parts and can access many parts of the store. Previously we opted to cover all functions inside queries by unit tests so we don't have to mock everything out.

Reply:
But actually, nevermind :D

I see that that's not consistently applied.

This is a personal nitpick.
I tend to prefer a function with a discriptive name over multiple imperative steps that are preceeded by a comment explaining what the steps do.